src/commands/flags.ts

```diff
@@ -2240,7 +2241,25 @@ export class Flags {
       describe: 'Used to specify desired number of consensus nodes for pre-genesis deployments',
       type: 'number',
     },
-    prompt: undefined,
+    prompt: async function promptAmount(task: SoloListrTaskWrapper<AnyListrContext>, input: number): Promise<number> {
```

Comment:
```suggestion
    prompt: async function promptNumberOfConsensusNodes(task: SoloListrTaskWrapper<AnyListrContext>, input: number): Promise<number> {
```
There is actually another flag titled `amount`, changing the name to avoid confusion.